src/App.scss

```diff
@@ -1,3 +1,9 @@
+// Your variable overrides
+$headings-margin-bottom: 1rem;
```

Comment:
The override should go in index.scss which is the top level SCSS file. App.scss is meant to be the styling for App.scss (which also means `footer-link` inside App.scss probably should be somewhere else, but we can leave that for now)

Reply:
Taken!